ghcide/src/Development/IDE/Core/Actions.hs

```diff
@@ -61,9 +61,24 @@ getAtPoint file pos = runMaybeT $ do
   (hf, mapping) <- useE GetHieAst file
   env <- hscEnv . fst <$> useE GhcSession file
   dkMap <- lift $ maybe (DKMap mempty mempty) fst <$> runMaybeT (useE GetDocMap file)
+  tcGblEnv <- tmrTypechecked . fst <$> useE TypeCheck file
```

Comment:
Using `TypeCheck` here is bad. It will make ghcide hovers slower, and it will break the ability to get hover information at cold startup from the local cache. 

Now, if fixity was in a plugin, it would still create all the problems above since `his-plugin-api` collects all the results before sending the response. But one can:
1. Disable the fixity plugin
2. Implement partial responses

I think this is a blocker. Workarounds:

1. Extract back to a plugin
2. Introduce a new build rule for fixity information, and give it a persistent implementation with `addPersistentRule`. The persistent implementation can return some placeholder or you can cache the fixity info in hiedb if desired.